packages/parse/src/index.ts

```diff
@@ -1,7 +1,7 @@
 import yaml from "js-yaml";
 import { Release, VersionType } from "@changesets/types";
 
-const mdRegex = /\s*---([^]*?)\n\s*---\s*\n([^]*)/;
+const mdRegex = /\s*---([^]*?)\n\s*---(?:\s*\n([^]*))?$/;
```

Comment:
This version of the regexp doesn't allow for a valid~ input like `"---\n--- "`

We can fix this with:
```suggestion
const mdRegex = /\s*---([^]*?)\n\s*---(\s*(?:\n|$)[^]*)/;
```

By using this suggestion we can also revert the change in the `summary` declaration as this regexp doesn't use an optional capturing group so it will always end up using an empty string for a `roughSummary` match.

Reply:
Very nice catch! This is updated now